src/Network/Freddy.hs

```diff
@@ -0,0 +1,48 @@
+{-# OPTIONS -XOverloadedStrings #-}
+module Network.Freddy (connect, respondTo) where
+
+import Network.AMQP
+import Data.Text (Text, pack)
+import Data.ByteString.Lazy.Char8 (ByteString)
+import Network.Freddy.Result
+
+connect :: String -> Text -> Text -> Text -> IO Connection
+connect host vhost user password =
```

Comment:
I'd like to keep it consistent with other freddy implementations. This means dropping `vhost` though. also ok?